Week3/Day2/src/pipeline/extract_employee_data_from_json.py

```diff
@@ -0,0 +1,54 @@
+import json
+from database_connection import *;
+from psycopg2.extras import Json
+from archieveTable import archieveTable
+
+def extract_employee_data_from_json(filePath):
```

Comment:
Try to break down these large functions into smaller ones.

Reply:
ok dai.